docs/user_guide/configurations/logging_configuration.rst

```diff
@@ -193,9 +193,11 @@ LoggerNameFilter
 ----------------
 :class:`LoggerNameFilter<nvflare.fuel.utils.log_utils.LoggerNameFilter>` filters loggers based on a list of logger_names.
 Filters utilize the logger hierarchy, so any descendants of the specified names will also be allowed through the filter.
+By default, LoggerNameFilter is configured with allow_errors to allow all logs with level greater than INFO though even if they are not from a logger in logger_names.
```

Comment:
Or maybe more descriptive here?
Not sure if this is overkill:

We have a mechanism in the `LoggerNameFilter` configuration that allows logs with a level greater than INFO, even if they do not come from loggers listed in logger_names. This behavior effectively bypasses the logger name filter for higher-level logs. By default, this mechanism is enabled (`allow_high_level_logs_to_bypass=True`), meaning logs above the INFO level are allowed regardless of the logger name.

Reply:
thanks for the name change, this line needs to be updated as well::

"allow_errors" -> `allow_all_error_logs`